package.json

```diff
@@ -56,7 +56,7 @@
         "webpack-node-externals": "^1.7.2"
     },
     "dependencies": {
-        "i18next": "^15.0.5",
+        "i18next": "^15.0.5 || ^19.0.0",
```

Comment:
how about major versions in between?

Reply:
We are upgrading to the latest version of i18next(from 15) so other versions are irrelevant